doc/themes-control-extensions.md

```diff
@@ -4,13 +4,6 @@ uid: Uno.Themes.Control.Extensions
 
 # Control Extensions
 
-Starting on version 5.0.0 the `ControlExtensions` are available on the `Uno.Themes` namespace, so to use that in xaml you should add the following xmlns to your project:
-
-`xmlns:ut="using:Uno.Themes"`
-
-> [!INFO]
-> For more information about see our [migration docs](material-migration.md).
-

```

Comment:
why this information is removed here @kazo0 ?

Reply:
I didn't think it was needed here. As long as the doc is up to date with the relevant APIs then I think we should keep the upgrading info centralized in the upgrading docs.

Otherwise, this doc will become stale with info relating to 5.0 stuff by the time we are on 6.0+ stuff

Reply:
And it is already mentioned in the migration doc already just in case?

Reply:
great thanks :) !